ovirt-guest-agent-fedora/ovirt-container-hibernate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/nsenter --mount=/proc/1/ns/mnt -- /usr/share/ovirt-guest-agent/hibernate $@
```

Comment:
the way to solve is probably to add this to config.json.template:

"DESTDIR=$DESTDIR",

and then use from the container:

$DESTDIR/rootfs/usr/share/ovirt-guest-agent/hibernate

DESTDIR is replaced at installation time with the path to the container checkout (from the host)